fiftyone/utils/torch.py

```
@@ -520,8 +589,71 @@
+        field_mapping (None): the user-supplied dict mapping keys in
+            :attr:`required_keys` to field names of their dataset that contain
+            the required values
+        transform (None): a ``torchvision.transforms`` function to apply
+        raw_inputs (False): whether to feed the raw list of images to the model
+            rather than stacking them as a Torch tensor
+        using_half_precision (False): whether the model is using half precision
+        use_numpy (False): whether to use numpy arrays rather than PIL images
+            and Torch tensors when loading data
+    """
+
+    def __init__(
```

Pylint (3.3.7)

[refactor] Too many positional arguments (6/5)

(R0917)